server/resources/migrations/54_add_storage_sweeper.up.sql

```diff
@@ -0,0 +1,4 @@
+alter table app_files_to_sweep add column process_id text;
+alter table app_files_to_sweep add column updated_at timestamp;
```

Comment:
I would make this `timestamp with time zone not null default now()` and then you can add

```sql
create trigger update_updated_at_trigger
  before update on app_files_to_sweep for each row
  execute function update_updated_at_column();
```

That way you don't have to manually modify the updated_at timestap (the `update_updated_at_column` function is already defined)